testdata/baselines/reference/fourslash/findAllRef/AmbientShorthandFindAllRefs.baseline.jsonc

```diff
@@ -0,0 +1,12 @@
+// === findAllReferences ===
```

Comment:
This looks different from the original baseline file in Strada, right? Where in the code are we accounting for those differences such that tests don't fail?

Reply:
The findAllReferences tests aren't being run against the submodule yet, they are just a standard "has the baseline changed" test. For FAR specifically, in corsa we currently only compute the locations of references--so most of the stuff that is in corsa's baseline (the json structures) is just info that is no longer computed.